utils/setup.cfg

```diff
@@ -5,7 +5,9 @@
 universal=1
 
 [nosetests]
-tests=../,../roles/openshift_master_facts/test/,test/
+tests = ../test/unit,
+        .,
+        ../roles/openshift_master_facts/test,
```

Comment:
Could have been a separate PR... I realized we were missing 2 tests via `cd utils; make ci`.